TestSuite/pack.py

```diff
@@ -126,6 +135,9 @@ def create_script(
             image
         )
         self.scripts.append(script)
+
+        script_filename = name + '.yml'
+        self.added_files.add(self._scripts_path / name / script_filename)
```

Comment:
why do we need the added_files?

Reply:
the UpdateRN class expects this set in order to create the release notes. when you use create_integration() or create_pack(), it will be added to this set (can use it for other types, but for start those are enough), and then when you call update_release_notes() it uses this set to indicate created files and then "flushes" its content for the next use of the function.

Reply:
so why don't you create a function that will calculate those?
With the necessity to add each object to this list manually, the next developer will forget to do so. Create a function that returns all file paths insteads

Reply:
I don't add it manually, it's added in create_integration() and create_script(). In what other way can this be done?